### PR TITLE
configuration for a subject prefix for email

### PR DESCRIPTION
**Pull request by francescob**
Added an option to add a subject prefix to the emails sent by solid_errors. Useful when you have multiple apps/environments using solid errors and what to easily filter the email received

### CODE REVIEW

File: app/mailers/solid_errors/error_mailer.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def error_occurred(occurrence)
       @error = occurrence.error
 
       mail(
-        subject: "#{@error.severity_emoji} #{@error.exception_class}",
+        subject: "#{SolidErrors.subject_prefix} #{@error.severity_emoji} #{@error.exception_class}",
```
Comment (fractaledmind) [inline app/mailers/solid_errors/error_mailer.rb:9]: Handle an empty prefix

**Review by fractaledmind — CHANGES_REQUESTED** [review]
I like the idea, just one small change needed

**Comment by francescob** [comment]
Ok, now the empty prefix is handled.
i also have another idea that fits in the way we use solid_errors:
Make subject_prefix enabled by default, with a value of 
`[#{Rails.application.name.upcase][#Rails.application.env.upcase]`.
Disable it with config.solid_errors.subject_prefix = "" or override it with your own custom string
What do you think about that?

**Review by fractaledmind — APPROVED** [review]
LGTM

**Comment by fractaledmind** [comment]
> Ok, now the empty prefix is handled. i also have another idea that fits in the way we use solid_errors: Make subject_prefix enabled by default, with a value of `[#{Rails.application.name.upcase][#Rails.application.env.upcase]`. Disable it with config.solid_errors.subject_prefix = "" or override it with your own custom string What do you think about that?

Making this the default would be a breaking change. I would rather introduce such a change slowly. So, let's start by providing that as the example in the README for this feature.

**Comment by francescob** [comment]
Ok i've renamed the option for conformity with other email settings and updated the README

File: README.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ You can configure Solid Errors via the Rails configuration object, under the `so
 * `sends_email` - Whether or not to send emails when an error occurs. See [Email notifications](#email-notifications) for more information.
 * `email_from` - The email address to send a notification from. See [Email notifications](#email-notifications) for more information.
 * `email_to` - The email address(es) to send a notification to. See [Email notifications](#email-notifications) for more information.
-
+* `email_subject_prefix` - The subject prefix to use for email notifications. See [Email notifications](#email-notifications) for more information.
```
[the remark this answers is not in the document]
Comment (fractaledmind) [reply README.md]: Still missing the newline before the next section title

**Review by fractaledmind — APPROVED** [review]
Lovely! Thank you for this

**Pull request merged**